playbooks/adhoc/uninstall.yml

```diff
@@ -39,7 +39,7 @@
         - atomic-openshift-master-api
         - atomic-openshift-master-controllers
         - atomic-openshift-node
-        - etcd
+        - "{% if is_atomic %}etcd_container{% else %}etcd{% endif %}"
```

Comment:
Thanks for your ansible tip.

@detiber etcd could be a critical service on atomic and it could be used by other docker containers. Should we assume that this playbook have to be used only on dedicated host for openshift ?
If you validate, I will change my PR to keep etcd and separate etcd_container.